tests/unit/histogram.cpp

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <gtest/gtest.h>
+
+#include "utils/histogram.hpp"
+#include "utils/logging.hpp"
+
+TEST(Histogram, BasicFunctionality) {
+  memgraph::utils::Histogram histo{};
+
+  for (int i = 0; i < 9000; i++) {
+    histo.Measure(10);
+  }
+  for (int i = 0; i < 900; i++) {
+    histo.Measure(25);
+  }
+  for (int i = 0; i < 90; i++) {
+    histo.Measure(33);
+  }
+  for (int i = 0; i < 9; i++) {
+    histo.Measure(47);
+  }
+  histo.Measure(500);
+
+  MG_ASSERT(histo.Percentile(0.0) == 10);
+  MG_ASSERT(histo.Percentile(99.0) == 25);
+  MG_ASSERT(histo.Percentile(99.89) == 32);
+  MG_ASSERT(histo.Percentile(99.99) == 46);
+  MG_ASSERT(histo.Percentile(100.0) == 500, "actual: {}", histo.Percentile(100.0));
```

Comment:
Please use `EXPECT_EQ` or `ASSERT_EQ` not `MG_ASSERT`. E.g.: they print the values when an assertion fails and GoogleTest provide a nice log when something fails.

Reply:
I've changed these to use ASSERT_EQ